docs/zz_generated.applicationInstallation.ce.yaml

```diff
@@ -26,7 +26,7 @@ spec:
     # Should be a valid lowercase RFC1123 domain name
     name: my-namespace
   # Values specify values overrides that are passed to helm templating. Comments are not preserved.
-  # Deprecated: Use ValuesBlock instead.
+  # Deprecated: Use ValuesBlock instead. This field was deprecated in KKP 2.25 and will be removed in KKP 2.27+.
```

Comment:
Note for reviewer:

If we are introducing the removal notices with 2.26 maybe we could move the removal period further by one release to KKP 2.28+. But I think that's not required, don't have strong opinions there though.